### PR TITLE
Death msgs

### PR DESCRIPTION
**Pull request by Cr0wb4r**
Added a few death messages for both reveal and no-reveal.

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
         "Congratulations! You're the 1337th person to use {cmd_char}ping. You win a goat!",
         "PING! {nick}",
         "I'm sorry Dave, I'm afraid I can't do that.",
-        "Give me a ping, Vasily. One ping only, please."
+        "Give me a ping, Vasily. One ping only, please.",
+        "WOULD YOU PEOPLE LEAVE ME ALONE AND STOP DOING THAT ?!?! Sheesh."
```
Comment (Vgr255) [inline messages/en.json]: Less caps per message please.

File: messages/en.json [hunk 2]
```diff
@@ -45,7 +46,9 @@
         "Despite protests, the mob drags their victim to the hanging tree. \u0002{0}\u0002 succumbs to the will of the horde, and is hanged. The villagers have killed a{1} \u0002{2}\u0002.",
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows. Once the twitching stops, it is discovered that the village lynched a{1} \u0002{2}\u0002.",
         "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
-        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early."
+        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early.",
+        "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered in chunks of dead \u0002{2}\u0002.",
+        "The villager decide to toss \u0002{0}\u0002 a{1} \u0002{2}\u0002 into the local volcano, effectively melting them."
```
Comment (jacob1) [inline messages/en.json]: missing a comma in both volcano messages, and also the spelunking messages below

File: messages/en.json [hunk 3]
```diff
@@ -72,7 +77,11 @@
         "A pack of wild animals sets upon \u0002{0}\u0002. Soon the \u0002{2}\u0002 is only a pile of bones and a lump in the beasts' stomachs.",
         "\u0002{0}\u0002, a{1} \u0002{2}\u0002, fell off the roof of their house and is now dead.",
         "\u0002{0}\u0002 is crushed to death by a falling tree. The villagers desperately try to save the \u0002{2}\u0002, but it is too late.",
-        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory. The survivors bury the \u0002{2}\u0002's ashes."
+        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory. The survivors bury the \u0002{2}\u0002's ashes.",
+        "\u0002{0}\u0002 played to much Russian roulette with their so called friends, and lost. The villagers bury the \u0002{2}\u0002's left overs.",
```
Comment (jacob1) [inline messages/en.json]: *too
*leftovers ):<

File: messages/en.json [hunk 4]
```diff
@@ -60,7 +63,9 @@
         "Despite protests, the mob drags their victim to the hanging tree. \u0002{0}\u0002 succumbs to the will of the horde, and is hanged.",
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows.",
         "Before the rope is pulled, \u0002{0}\u0002 pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
-        "Before the rope is pulled, \u0002{0}\u0002 throws a grenade at the mob. The grenade explodes early."
+        "Before the rope is pulled, \u0002{0}\u0002 throws a grenade at the mob. The grenade explodes early.",
+        "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered in chunks of dead [redacted].",
```
Comment (jacob1) [inline messages/en.json]: [redacted] looks weird here, maybe replace it with something else like 'villager' or 'their dead comrade' or something.

File: messages/en.json [hunk 5]
```diff
@@ -72,7 +77,11 @@
         "A pack of wild animals sets upon \u0002{0}\u0002. Soon the \u0002{2}\u0002 is only a pile of bones and a lump in the beasts' stomachs.",
         "\u0002{0}\u0002, a{1} \u0002{2}\u0002, fell off the roof of their house and is now dead.",
         "\u0002{0}\u0002 is crushed to death by a falling tree. The villagers desperately try to save the \u0002{2}\u0002, but it is too late.",
-        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory. The survivors bury the \u0002{2}\u0002's ashes."
+        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory. The survivors bury the \u0002{2}\u0002's ashes.",
+        "\u0002{0}\u0002 played to much Russian roulette with their so called friends, and lost. The villagers bury the \u0002{2}\u0002's left overs.",
+        "\u0002{0}\u0002 poked a bear in the eye with a stick. The villagers burry what's left of the \u0002{2}\u0002's body.",
```
Comment (jacob1) [inline messages/en.json]: *bury

Comment (Cr0wb4r) [inline messages/en.json]: Thanks, fixed that in both the reveal and no-reveal

File: messages/en.json (hunk 2 above)
Comment (Cr0wb4r) [inline messages/en.json]: I see where I missed it in the first group, but is is needed in the second? (no-reveal)

File: messages/en.json [hunk 6]
```diff
@@ -45,7 +46,9 @@
         "Despite protests, the mob drags their victim to the hanging tree. \u0002{0}\u0002 succumbs to the will of the horde, and is hanged. The villagers have killed a{1} \u0002{2}\u0002.",
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows. Once the twitching stops, it is discovered that the village lynched a{1} \u0002{2}\u0002.",
         "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
-        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early."
+        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early.",
+        "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered in chunks of dead \u0002{2}\u0002.",
```
Comment (Vgr255) [inline messages/en.json]: "With the remains of a late __seer__" for example, or "With the remains of the late comrade"

**Comment by Diitto** [comment]
"left overs" remains would be nicer

File: messages/en.json (hunk 6 above)
Comment (Cr0wb4r) [inline messages/en.json]: What about something along the lines of:
"covered in what is left of the late seer/wolf/villager"

**Comment by lutoma** [comment]
I like the messages. :+1: 

(This message may or may not be mostly or partly sent for the purpose of spamming @mj94 with another email notification)

File: messages/en.json [hunk 7]
```diff
@@ -84,7 +93,12 @@
         "A pack of wild animals sets upon \u0002{0}\u0002. Soon they are only a pile of bones and a lump in the beasts' stomachs.",
         "\u0002{0}\u0002 fell off the roof of their house and is now dead.",
         "\u0002{0}\u0002 is crushed to death by a falling tree. The villagers desperately try to save them, but it is too late.",
-        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory."
+        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory.",
+        "\u0002{0}\u0002 played too much Russian roulette with their so called friends, and lost. The villagers bury the leftovers.",
```
Comment (skizzerz) [inline messages/en.json]: entire thing is really clunky and awkward, needs rewording

File: messages/en.json [hunk 8]
```diff
@@ -84,7 +93,12 @@
...
+        "\u0002{0}\u0002 poked a bear in the eye with a stick. The villagers bury what's left of the body.",
+        "\u0002{0}\u0002 tried to smoke dynamite. The resulting explosion has left a crater in their place.",
+        "\u0002{0}\u0002 went spelunking and never made it back."
+
```
Comment (skizzerz) [inline messages/en.json]: extraneous blank line

File: messages/en.json (hunk 7 above)
Comment (Cr0wb4r) [inline messages/en.json]: Which parts? The whole thing?
What about something like "played Russian roulette with their so called friends, and lost." Instead?

Comment (nyuszika7h) [inline messages/en.json]: so-called*

File: messages/en.json (hunk 8 above)
Comment (nyuszika7h) [inline messages/en.json]: &nbsp;

File: messages/en.json (hunk 7 above)
Comment (lutoma) [inline messages/en.json]: I'd drop the "so-called". Feels clunky.

Comment (laneatomic) [inline messages/en.json]: Remains, not leftovers. They're not food. Also, +1 to drop the so-called.

File: messages/en.json [hunk 9]
```diff
@@ -72,7 +77,11 @@
         "A pack of wild animals sets upon \u0002{0}\u0002. Soon the \u0002{2}\u0002 is only a pile of bones and a lump in the beasts' stomachs.",
         "\u0002{0}\u0002, a{1} \u0002{2}\u0002, fell off the roof of their house and is now dead.",
         "\u0002{0}\u0002 is crushed to death by a falling tree. The villagers desperately try to save the \u0002{2}\u0002, but it is too late.",
-        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory. The survivors bury the \u0002{2}\u0002's ashes."
+        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory. The survivors bury the \u0002{2}\u0002's ashes.",
+        "\u0002{0}\u0002 played Russian roulette, and lost. The villagers bury the \u0002{2}\u0002's remains.",
+        "\u0002{0}\u0002, a{1} \u0002{2}\u0002, poked a bear in the eye with a stick and has paid the price.",
+        "\u0002{0}\u0002 tried to smoke dynamite. The resulting explosion has left a crater in place of the \u0002{2}\u0002.",
+        "\u0002{0}\u0002, the \u0002{2}\u0002, went spelunking and never made it back."
```
Comment (laneatomic) [inline messages/en.json]: a/an instead of the.

**Pull request merged**